ui/shared/components/panel/search/constants.js

```diff
@@ -324,6 +324,12 @@ export const FREQUENCIES = [
     homHemi: false,
     labelHelp: 'Filter by allele count (AC) or allele frequency (AF) in TOPMed',
   },
+  {
+    name: 'gnomad_genome_svs',
```

Comment:
as mentioned above, this should just be `gnomad_svs`. This  data in gnomad isn't genome specific so you should take out the word genome from the description as well